administrator/components/com_associations/models/associations.php

```diff
@@ -0,0 +1,298 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_associations
+ *
+ * @copyright   Copyright (C) 2005 - 2016 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+/**
+ * Methods supporting a list of article records.
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class AssociationsModelAssociations extends JModelList
+{
+	/**
+	 * Constructor.
+	 *
+	 * @param   array  $config  An optional associative array of configuration settings.
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 * @see     JController
+	 */
+	public function __construct($config = array())
+	{
+		if (empty($config['filter_fields']))
+		{
+			$config['filter_fields'] = array(
+				'id', 'a.id',
+				'a.name',
```

Comment:
you need to put the rder fields here exactly like the xml and the table headedings.

So if you use the `title` alias you need to put here `title` too instead of `a.name`